data/i18n/en/en.toml

```diff
@@ -355,6 +355,16 @@ other = "Before you begin"
 [previous_patches]
 other = "Patch Releases:"
 
+[release_binary_info]
+version_option = "Stable Versions"
+os_option = "Operating Systems"
+arch_option = "Architectures"
+version = "Version"
+os = "Operating System"
+arch = "Architecture"
+download_binary = "Download Binary"
+copy_link = "Copy Link"
```

Comment:
Neat as this is, I would start off using `other` for each entry. We could switch to a new approach like this but that should be as a _separate_ change.